cmd/mycelo/genesis_commands.go

```diff
@@ -135,6 +136,8 @@ func envFromTemplate(ctx *cli.Context, workdir string) (*env.Environment, *genes
 		} else {
 			gingerbreadBlock = big.NewInt(gingerbreadBlockNumber)
 		}
+	} else {
+		gingerbreadBlock = common.Big1
```

Comment:
Why 1? I would have expected 0?

Reply:
Then I run into this: https://github.com/celo-org/celo-blockchain/blob/459662f782d27ece531e967eb1cd7abb1ad8457c/mycelo/genesis/genesis.go#L28-L34

Maybe there's a better solution though.